pkg/controller/kfdef/kfdef_controller.go

```diff
@@ -250,6 +251,17 @@ func kfLoadConfig(instance *kfdefv1.KfDef, reqLogger logr.Logger, action string)
 		reqLogger.Info("Failed to write config.yaml ", err)
 		return nil, err
 	}
+        
+        // This is temp. fix to allow users to install multiple kfdefs
+        if action == "apply" {
+            // Delete kustomize folder
+            reqLogger.Info("Deleting kustomize Folder")
```

Comment:
I would recommend to move the deletion code into `kfApply` function and check for the `/tmp/kustomize` existence so that we avoid printing the error on first apply